### ISSUE TITLE
Fix incorrect description of Hestenes inner product in examples

### ISSUE THREAD

**Issue by utensil** [pygae/galgebra#139]
In https://nbviewer.jupyter.org/github/pygae/galgebra/blob/master/examples/ipython/inner_product.ipynb : 

![image](https://user-images.githubusercontent.com/64258/70333981-7faa4580-187f-11ea-9859-68cf385b9cdc.png)

It's inconsistent with implementation and the original literature:

![image](https://user-images.githubusercontent.com/64258/70334057-a36d8b80-187f-11ea-8b44-ef1e409f6afa.png)

This should be corrected. And it's better to add some description of all three definitions:

![image](https://user-images.githubusercontent.com/64258/70334133-cdbf4900-187f-11ea-93cf-473bc635d212.png)

from Geometric Algebra with Applications in Science and Engineering .

**Comment by eric-wieser** [comment]
> It's inconsistent with implementation and the original literature

How so? Looks identical to the implementation and literature to me, if you start with the assumption that r and s are non-negative

**Comment by utensil** [comment]
![image](https://user-images.githubusercontent.com/64258/70335087-95206f00-1881-11ea-8ea4-1e400609b5fe.png)
 
This is particularly wrong (my own mistake). And the formula fails to mention how it should be like for arbitrary multivectors (i.e. the summing).

**Comment by eric-wieser** [comment]
Apart from the out of place "of", that sentence is true, isn't it?

> And the formula fails to mention how it should be like for arbitrary multivectors

That's fair.

It might be worth adding the mathematical description to the docstring of `hestenes_dot`

**Comment by utensil** [comment]
> Apart from the out of place "of", that sentence is true, isn't it?

No. It intended to describe how it works, not a consequence of how it works.
